### DISCUSSION TITLE
Is it possible to add video with an alpha layer to Hubs?

### DISCUSSION THREAD

**Discussion by emclaren** [mozilla/hubs#2719] [General Questions & Discussion] [answered]
Does anybody know a way of adding video with an alpha layer, or is that not possible?

(note this is a question copied over from Discord)

**Comment by emclaren** [comment, marked as answer]
To my knowledge, at the moment it's not currently possible to add a video with a transparency layer to Hubs. 

Images with transparent alpha channels are supported (e.g. png/gif)

**Comment by arpu** [comment]
does anyone have a good video sample with alpha layer?

**Reply by emclaren** [reply]
I just did a search online to find one for testing. An example of one can be found here: https://codepen.io/xy_r/pen/JjodVvB

**Comment by emclaren** [comment]
Due to popular demand I created an issue for this here : https://github.com/mozilla/hubs/issues/3007
